src/components/picture-in-picture/picture-in-picture.js

```diff
@@ -0,0 +1,93 @@
+//@flow
+import style from '../../styles/style.scss';
+import {h} from 'preact';
+import {Localizer, Text} from 'preact-i18n';
+import {default as Icon, IconType} from '../icon';
+import {KeyMap} from '../../utils/key-map';
+import BaseComponent from '../base';
+
+/**
+ * PictureInPicture component
+ *
+ * @class PictureInPicture
+ * @extends {BaseComponent}
+ */
+class PictureInPicture extends BaseComponent {
```

Comment:
Don't create the component at all if isPictureInPictureSupported() returns false.

Reply:
@OrenMe

Reply:
?

Reply:
@dan-ziv we will use this component rendering like this. 
virtual DOM knows to render or not by receiving undefined.